code_map.py

```diff
@@ -290,6 +298,21 @@ def go():
 # -----------------
 
 
+def focus_source_code():
+    if CodeMapListener.active_view:
+        w = win()
+        w.focus_view(CodeMapListener.active_view)
+
+# -----------------
+
+
+def scroll(v):
+    line = v.line(v.sel()[0].a)
+    v.show_at_center(line.a)
+
+# -----------------
+
```

Comment:
moved focus_source_code() here, so no need to repeat. also moved scroll() here because it didn't scroll left when double-cliccking on map, only when navigating